packages/ia-components/sandbox/tiles/TileComponent.jsx

```diff
@@ -86,6 +89,21 @@ export default class TileComponent extends IAReactComponent {
     render() {
         return (
             <div className={this.state.classes.join(' ')} data-id={this.state.identifier}  key={this.state.identifier}>
+                {/*-- Add in experimental crawl notification for dweb-mirror, if member.crawl=undefined then ignored --*/}
+                {(!(this.state.crawl.level || this.state.downloaded)) ? null : // Only show this bug if crawling
+                    <div className="item-crawl">
+                        <div className="item-crawl-img">
+                            { this.state.crawl.level === "details"
```

Comment:
Since this conditional block outputs nearly identical markup with each condition, we could create a stateless functional component for the markup and pass in the fill color and crawl level.

Reply:
Again - please don't make suggestions that are just your prefered way of coding, feel free to spend time rewriting afterwards if you wish